src/index.ts

```diff
@@ -154,22 +196,51 @@ function verifyCurrentCombination() {
   addTargetListener(currentTargets, COLORS);
 }
 
+function cancelPopup(){
```

Comment:
No need to create function , you could directly do smthing like this


```ts
parameters.onclick = ()=>{
  parametersPopup.style.display = 'flex';
  parameters.style.display = 'none';
};

```


It is more readable i think. 

Same for showPopup.

(ps: the name of the function wasn't very explicit, it would have been better if you use `showParamsPopup` instead bcs there is other popUp than params in the game )

Reply:
done, and true, surely more readable